package.json

```diff
@@ -173,7 +173,7 @@
     "camelcase": "^6.1.0",
     "colors": "1.4.0",
     "css-loader": "4.3.0",
-    "cypress": "5.2.0",
+    "cypress": "10.1.0",
```

Comment:
nit. can we upgrade to 10.2 since thats the latest as of now? It offers performance improvements on M1 machines and some of our team members are starting to get them.